src/skai/model/data_loader_test.py

```diff
@@ -38,6 +38,7 @@ def _make_serialized_image(size: int, pixel_value: int) -> bytes:
 
 def _make_example(
     example_id: str,
```

Comment:
Let's get rid of this parameter since it's no longer relevant.

Reply:
Removed together with "string label" which is not used as well.